Basic Updates

christiantonti opened the pull request:
-Added loading spinner while awaiting Heroku response (there is a 20-30 second API delay after Heroku gets no requests for 30 minutes)
-Centered table header text
-Added title bar and short description
-Added general padding and some bootstrap styling

Further updates to come; maybe coloration, a user lookup, or a login screen.

christiantonti committed:
Basic Updates

-Added loading spinner while awaiting Heroku response
-Centered table header text
-Added title bar and short description
-Added general padding and some bootstrap styling

Further updates to come.

evinaik commented on index.html:
```diff
@@ -11,9 +11,18 @@
+        <span class="sr-only">Loading...</span>
+      </div>  
+    </div>
+    <table id="table" class="table table-striped" style="display: none;">
```
Why is this style inline?

christiantonti replied:
Using the bootstrap display none class resulted in the table not appearing upon using jQuery's fadeIn() function, so an inline worked best. Obviously a style tag in the header would work as well but I maximized bootstrap use.

evinaik commented on index.html:
```diff
@@ -11,9 +11,18 @@
+      <h5 class="text-center">A log of all strikes added and removed and their reasons why.</h5>  
+    </div>
+    <div id="loading-message" class="text-center mt-2">
+      <h5 class="py-2">Loading Table...</h5>
```
Can you provide some context about these py-X classes? I'm not sure what they are, though I'm interested.

christiantonti replied:
Bootstrap has spacing classes for margin and padding. 'p' stands for padding, 'y' stands for top and bottom (y-axis), and the number is a multiplier to the $spacer bootstrap sass variable.
See: https://getbootstrap.com/docs/4.3/utilities/spacing/

evinaik commented:
Great speed on this. For UI PR's, you generally include a before/after screenshot (just after is okay), for a sanity check. Once you add that and comments, I'll approve.

evinaik reviewed (changes requested):
See comments.

christiantonti commented:
Changes and a UI before/after photo will be up by tonight, along with some comments in the code for clarity.

christiantonti committed:
Update index.html

- Added some descriptive comments
- Moved inline styles to style tag in head

christiantonti commented:
Added descriptive comments to some areas for clarity. Also moved inline styles to the <style> tag in the head. Attached are a before and after photo of the UI.
Before:
![image](https://user-images.githubusercontent.com/50304532/66704739-fb0ed100-ecec-11e9-82a3-174071371689.png)
After (note that the loading spinner is not shown here):
![image](https://user-images.githubusercontent.com/50304532/66704751-15e14580-eced-11e9-905b-44cc40de32ba.png)

evinaik commented:
LGTM
